server/data-base/inventory.json

```diff
@@ -0,0 +1,47 @@
+{
+  "lastMigrationId": "n/a",
```

Comment:
can we make this an actual date from the past? `"2024-06-11T21:00:00.000Z"` would work